### ISSUE TITLE
Start new experiment when user presses play (Trac #48)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#52]
If the user performs steps in realtime prior to pressing play (i.e., running a protocol), the realtime steps show up in the experiment log. It would be preferable to start a new experiment when the user presses play, which would differentiate between the realtime and automated parts of the experiment.

Migrated from http://microfluidics.utoronto.ca/ticket/48
```json
{
    "status": "closed", 
    "changetime": "2015-01-13T08:37:18", 
    "description": "If the user performs steps in realtime prior to pressing play (i.e., running a protocol), the realtime steps show up in the experiment log. It would be preferable to start a new experiment when the user presses play, which would differentiate between the realtime and automated parts of the experiment.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1421138238797413", 
    "component": "core", 
    "summary": "Start new experiment when user presses play", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-02-20T03:41:37", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120828T16:16:39: ryan commented:

> As of v0.1-629-gba11ee3, steps run in real-time mode don't show up in the experiment log. Steps are only added to the experiment log when a protocol is running. So this ticket should be changed to:
> 
> 1. Record data for steps run in real-time mode (save the experiment log if the id/device change or the program exits).
> 2. Start a new log in response to an on_protocol_run signal if real-time steps exist in the experiment log.

**Comment by cfobel** [comment]
Trac update at 20140312T20:02:01: ryan changed milestone from "1.0" to "1.1"

**Comment by cfobel** [comment]
Trac update at 20140313T20:00:48: ryan changed milestone from "1.1" to ""

**Comment by cfobel** [comment]
Trac update at 20150105T18:08:41: cfobel changed milestone from "" to "Microdrop 1.0"

**Comment by cfobel** [comment]
Trac update at 20150113T08:37:18: 
* ryan commented:

> Fixed in commit changeset 2ff927c142c84a4d1dac26c3070b955aaa1d89e1.


* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"
